doc/classes/PolygonPathFinder.xml

```diff
@@ -42,6 +42,17 @@
 			<return type="bool" />
 			<param index="0" name="point" type="Vector2" />
 			<description>
+				Returns [code]true[/code] if [param point] falls inside the polygon area, returns [code]false[/code] otherwise. 
```

Comment:
Nitpick:
```suggestion
				Returns [code]true[/code] if [param point] falls inside the polygon area, otherwise returns [code]false[/code]. 
```
Alternatively, I quote an Italian idiom: "_Se non è zuppa è pan bagnato_". Between two choices, if one is wrong, the other must be correct. If it's not `true`, it has to be `false`.
```suggestion
				Returns [code]true[/code] if [param point] falls inside the polygon area. 
```

Reply:
Forgive me. The above fix includes a trailing space which makes the check fail.

Reply:
> Forgive me. The above fix includes a trailing space which makes the check fail.

Fixed with a new commit, thanks.